nicegui/run.py

```diff
@@ -88,7 +87,7 @@ def run(*,
     os.environ['NICEGUI_URL'] = f'http://{host}:{port}'
 
     if show:
-        webbrowser.open(f'http://{host if host != "0.0.0.0" else "127.0.0.1"}:{port}/')
+        helpers.schedule_browser(host, port)
```

Comment:
I like that you moved the code into helpers.py. This way we can use/adapt it for https://github.com/zauberzeug/nicegui/blob/main/nicegui/native_mode.py to fix #607.